internal/controller/api/v1alpha1/installenv_types.go

```diff
@@ -21,6 +21,49 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
```

Comment:
Putting it here because it's the highest line that I can put it on...doesn't having all of these types in `internal/...` prevent anyone else from being able to import and use these types down the line?

Reply:
You are right. we probably need to move the types outside of internal, will be done in a different PR.

Reply:
We'll track this in [MGMT-4197]( https://issues.redhat.com/browse/MGMT-4197)